.circleci/test-deploy.yml

```diff
@@ -118,9 +118,36 @@ jobs:
           configure_git: false
           image_context: ./fixtures/docker
           image_name: busybox
-          image_tag: test
+          image_tag: tag_only
```

Comment:
🐼 

@zyassine I would have enforced publishing the main `build command` with latest + CircleCI SHA, while publishing with a single tag all other integration tests (to avoid overwriting the `latest` tag).